lighthouse-core/runner.js

```diff
@@ -125,7 +125,7 @@ class Runner {
         environment: {
           networkUserAgent: artifacts.NetworkUserAgent,
           hostUserAgent: artifacts.HostUserAgent,
-          benchmarkIndex: artifacts.BenchmarkIndex,
+          benchmarkIndex: parseInt(artifacts.BenchmarkIndex.toString()),
```

Comment:
`Math.round()` :)

But I think we want to do this either in `Driver.getBenchmarkIndex()` or in `ultradumbBenchmark` itself